chart/values.yaml

```diff
@@ -95,6 +95,15 @@ affinity: {}
 # see: https://github.com/caas-team/sparrow/blob/main/docs/sparrow_run.md
 extraArgs: {}
 
+#
+
+envFromSecrets:
```

Comment:
add a short comment how to use this/link this https://github.com/caas-team/sparrow/blob/main/README.md#configuration